src/status.js

```diff
@@ -0,0 +1,41 @@
+export default class Status {
+  constructor() {
+    if (localStorage.getItem('taskList') === null) {
+      this.taskList = {
+        0: {
+          description: 'task1',
+          completed: false,
+          index: 0,
+        },
+        1: {
+          description: 'task2',
+          completed: false,
+          index: 1,
+        },
+        2: {
+          description: 'task3',
+          completed: false,
+          index: 2,
+        },
+      };
+    } else {
+      this.taskList = JSON.parse(localStorage.getItem('taskList'));
+    }
+  }
+
+    saveStorage = () => {
+      localStorage.setItem('taskList', JSON.stringify(this.taskList));
+    }
+
+    mark = (index) => {
+      const task = this.taskList[index];
+      task.completed = true;
+      this.saveStorage();
+    }
+
+    unmark = (index) => {
```

Comment:
- [ ] Please add a event listener (`change`) to the checkbox to track the changes. The task object's values for `completed` is updated upon user actions.

Reply:
![Captura de pantalla de 2021-12-04 12-14-41](https://user-images.githubusercontent.com/73128809/144720360-b73d7617-f57e-4e1a-b2af-0106b1d0ba65.png)

I added the event listener change in the index script file.